docs/intro/tutorial.rst

```diff
@@ -159,25 +130,51 @@ similar to this::
     2016-09-01 16:51:29 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/2/> (referer: None)
     2016-09-01 16:51:29 [scrapy] INFO: Closing spider (finished)
 
-.. note::
-    At the end you can see a log line for each URL defined in ``start_urls``.
-    Because these URLs are the starting ones, they have no referrers, which is
-    shown at the end of the log line, where it says ``(referer: None)``.
+Now, check the files in the current directory. You should notice that two new
+files have been created: *quotes-1.html* and *quotes-2.html*, with the content
+for the respective URLs, as our ``parse`` method instructs.
+
+.. note:: If you are wondering why we haven't parsed the HTML yet, hold
+  on, we will cover that soon.
 
-Now, check the files in the current directory. You should notice two new files
-have been created: *quotes-1.html* and *quotes-2.html*, with the content for the respective
-URLs, as our ``parse`` method instructs.
 
 What just happened under the hood?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Scrapy schedules the :class:`scrapy.Request <scrapy.http.Request>` objects
+returned by the ``start_requests`` method of the Spider. Upon receiving
+a response for each one, it instantiates :class:`scrapy.http.Response`
+objects and calls the ``parse`` callback method passing the response as
+argument.
+
 
-Scrapy creates :class:`scrapy.Request <scrapy.http.Request>` objects
-for each URL in the ``start_urls`` attribute of the Spider, and assigns
-them the ``parse`` method of the spider as their callback function.
+Simplifying your spider
```

Comment:
I'm not sure if using start_urls can be described as simplifying, because there are actually more things entangled behind the scenes (i.e., this second version is shorter, but not necessarily simpler).

I think it's best to present as a short version, saying something like: "This pattern of starting from a few hardcoded URLs is so common that Scrapy has a shortcut for it: you can define a start_urls class attribute w/ a list of URLs, which will be used by the default implementation of Spider.start_requests to create the initial requests"

Makes sense?